includes/emails/email-summary/class-edd-email-summary.php

```diff
@@ -0,0 +1,791 @@
+	 */
+	private function build_relative_markup( $relative_data ) {
+		$arrow  = $relative_data['positive_change'] ? 'icon-arrow-up.png' : 'icon-arrow-down.png';
+		$output = __( 'No data to compare', 'easy-digital-downloads' );
+		if ( $relative_data['no_change'] ) {
+			$output = __( 'No Change', 'easy-digital-downloads' );
+		} elseif ( $relative_data['comparable'] ) {
+			$output = $relative_data['formatted_percentage_change'] . '%';
+		}
+		ob_start();
+		?>
+		<?php if ( $relative_data['comparable'] ) : ?>
+			<img src="<?php echo esc_url( EDD_PLUGIN_URL . '/assets/images/icons/' . $arrow ); ?>" width="12" height="10" style="outline: none; text-decoration: none; -ms-interpolation-mode: bicubic; width: 12px; height: 10px; max-width: 100%; clear: both; vertical-align: text-top;">
+		<?php endif; ?>
+		<span style="padding-left: 1px;">
+			<?php echo esc_html( $output ); ?>
+		</span>
+		<?php
+		$relative_markup = ob_get_clean();
+
+		return $relative_markup;
+	}
+
+	/**
+	 * Build email template.
+	 *
+	 * @since 3.1
+	 *
+	 * @param array|bool $blurb Structured blurb data.
+	 *
+	 * @return string|bool The string of the email template or false if the email template couldn't be built.
+	 */
+	public function build_email_template( $blurb = false ) {
+		$dataset        = $this->get_report_dataset();
+		$date_range     = $this->get_report_date_range();
+		$site_url       = $this->get_site_url();
+		$wp_date_format = get_option( 'date_format' );
+		$period_name    = ( 'monthly' === $this->email_options['email_summary_frequency'] ) ? __( 'month', 'easy-digital-downloads' ) : __( 'week', 'easy-digital-downloads' );
+		/* Translators: period name (e.g. week) */
+		$relative_text  = sprintf( __( 'vs previous %s', 'easy-digital-downloads' ), $period_name );
+
+		// If there were no sales, do not build an email template.
+		if ( 0 === $dataset['order_count'] ) {
+			return false;
+		}
+
+		ob_start();
+		?>
+		<!DOCTYPE html>
+		<html>
+			<head>
+				<title><?php echo esc_html( $this->get_email_subject() ); ?></title>
+				<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+				<meta name="viewport" content="width=device-width, initial-scale=1.0">
+				<!--[if !mso]><!-->
+				<meta http-equiv="X-UA-Compatible" content="IE=edge">
+				<!--<![endif]-->
+
+				<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="">
+				<link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300;400;600;700&display=swap" rel="stylesheet">
+			</head>
+			<style>
+				@media only screen and (max-width: 599px) {
+					.push-down-25 {
+						margin-bottom: 20px;
+					}
+					.pull-down-8 {
+						margin-top: 4px !important;
+					}
+					.pull-down-5 {
+						margin-top: 2.5px !important;
+					}
+					h1 {
+						font-size: 18px !important;
+						line-height: 24px !important;
+					}
+					h2 {
+						font-size: 12px !important;
+						line-height: 16px !important;
+					}
+					p {
+						font-size: 10px !important;
```

Comment:
@robincornett we cannot use anything other than PX b/c other units do not have full support in email clients. Important is necessary in order to make sure all clients (many of them browsers) don't stomp on our reports.

Since we aren't trying to 'play nice' with themes or anything this is completely in bounds in this type of scenario.

As for the font size @zkawesome let's try and bump it up a bit,  and try having the main content areas (tiles and table) match the sizes used in the desktop view please.